docs/agent/hostname-resolution.md

```diff
@@ -29,7 +29,7 @@ If you're affected by this change, we recommend that you take the following acti
 
 - Upgrading from Agent v5 to Agent v6.3: enable the `hostname_fqdn` option in the Agent v6 configuration to ensure that you will keep the same hostname.
```

Comment:
not related to your change, but the case described on this line applies to all versions v6.3 to v6.5, not only v6.3